lib/caiquenator_web/controllers/page_controller.ex

```diff
@@ -1,7 +1,11 @@
 defmodule CaiquenatorWeb.PageController do
   use CaiquenatorWeb, :controller
 
+  alias Caiquenator.Phrases
+
   def index(conn, _params) do
-    render conn, "index.html"
+    phrase = Phrases.get_random()
+
+    render(conn, "index.html", %{phrase: phrase})
```

Comment:
You don't need to worry about passing the `phrase` inside a map struct. Usually, we use a `keyword` list to pass a variable to a template 🎨 . WDYT?

```elixir
render(conn, "index.html", phrase: phrase)
```